Makefile

```diff
@@ -1,18 +1,34 @@
-BUILD_NUMBER := 1
-CP_VERSION := 3.3.0-SNAPSHOT
+# You can override vars like REPOSITORY in a local.make file
+-include local.make
```

Comment:
It works, but will leave it to other with more experience with make to comment naming and best practice in including other Makefiles.